pkg/index/platform_test.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package index
```

Comment:
The commit that moved this code can be reviewed with `git log -p --color-moved=dimmed-zebra fc890c7ba5640c2ba2087e0fe21b15260fa3f2a8^!`

Reply:
Looks like this just increased the amount of code to review.
I'm tempted to ask: Is there a way to reduce this PR? :D

Reply:
I extracted the respective commit.